src/data.py

```diff
@@ -92,6 +92,10 @@ def get_items(self) -> list:
         """Get all managed items."""
         return list(self._items.values())
 
+    def get_item_by_index(self, index: int):
+        """Get item given an index."""
+        return list(self._items.values())[index]
+
     def set_items(self, items: list):
```

Comment:
Use regular access instead, see line 138

Reply:
You are right, thanks, completely missed this.